ods_ci/tests/Resources/Files/llm/caikit_isvc.yaml

```diff
@@ -8,6 +8,9 @@ metadata:
   name: {{INFERENCE_SERVICE_NAME}}
 spec:
   predictor:
+    scaleMetric: concurrency
```

Comment:
shall we use a variable for this too? and we set to concurrency by default

Reply:
done